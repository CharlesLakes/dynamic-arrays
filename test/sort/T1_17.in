10
97 6 53 22 60 25 80 59 72 11