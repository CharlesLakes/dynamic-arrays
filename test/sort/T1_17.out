6 11 22 25 53 59 60 72 80 97